### ISSUE TITLE
Existing Bot Defense Profile not Assignable (no pointer)

### ISSUE THREAD

**Issue by xags** [F5Networks/f5-appsvcs-extension#193]
##### Do you already have an issue opened with F5 support?
Github Issues are consistently monitored by F5 staff, but should be considered as best effort only and you should not expect to receive the same level of response as provided by F5 Support. Please [open an case](https://support.f5.com/csp/article/K2633) with F5 if this is a critical issue.

##### ISSUE TYPE

 - Bug Report

##### AS3 BUILD/ VERSION

```
f5-appsvcs-3.15.0-6
```

##### BIGIP VERSION

```
Sys::Version
Main Package
  Product     BIG-IP
  Version     14.1.2.1
  Build       0.0.4
  Edition     Point Release 1
  Date        Tue Sep 17 15:44:32 PDT 2019

```

##### SUMMARY

```
It currently isn't possible to reference a bot-defense profile that already exists on a BIG-IP endpoint.
It is however possible to use the DOS_Profile class to create a new one.
It should be possible to reference an existing profile--IE: Use /Common/bot-defense
```

##### STEPS TO REPRODUCE

-Create a bot defense profile on an existing BIG-IP.
-Attempt to reference the profile in a new deceleration.

```
                    "profileDOS": {
                        "bigip": "/Common/bot_profile"
                    }
```

##### EXPECTED RESULTS

```
Have the ability to reference a bot profile and not create a new one.
```

##### ACTUAL RESULTS

```
Unable to reference bot profiles.
```

**Comment by dstokesf5** [comment]
Before BIG-IP v14.1, bot defense profiles were part of the DOS profile. We have an item in our internal product backlog (AUTOTOOL-600) to address this limitation on v14.1+

**Comment by dstokesf5** [comment]
This should be resolved in the 3.17.0 release. If you are still experiencing a problem, please re-open this issue or create a new one.
